.github/workflows/ci.yml

```
@@ -2,35 +1,26 @@
-      contents: read
-    steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - uses: denoland/setup-deno@5e01c016a857a4dbb5afe9d0f9733cd472cba985 # v1.5.1
-      - run: deno task lint
+      - name: Checkout your repository using git
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Install pnpm
+        uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+        with:
+          version: 9.14.4  # enovate: depName=pnpm
+      - name: Use Node.js ${{ matrix.node-version }}
```

actionlint (1.7.4)

property "node-version" is not defined in object type {}

(expression)